apps/site/pages/docs/building-sections/overrides/list-of-native-sections.mdx

```diff
@@ -93,10 +633,149 @@ Check out below a list of FastStore native sections, their components that can b
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Overridable components            | <ul className="doc-list"><li>[`ProductTitle`](https://developers.vtex.com/docs/guides/faststore/molecules-product-title)</li><li>[`DiscountBadge`](https://developers.vtex.com/docs/guides/faststore/molecules-discount-badge)</li><li>[`BuyButton`](https://developers.vtex.com/docs/guides/faststore/molecules-buy-button)</li><li>[`Icon`](https://developers.vtex.com/docs/guides/faststore/atoms-icon)</li><li>[`Price`](https://developers.vtex.com/docs/guides/faststore/atoms-price)</li><li>[`QuantitySelector`](https://developers.vtex.com/docs/guides/faststore/molecules-quantity-selector)</li><li>[`SkuSelector`](https://developers.vtex.com/docs/guides/faststore/molecules-sku-selector)</li><li>[`ShippingSimulation`](https://developers.vtex.com/docs/guides/faststore/organisms-shipping-simulation)</li><li>[`ImageGallery`](https://developers.vtex.com/docs/guides/faststore/organisms-image-gallery)</li><li>[`ImageGalleryViewer`](https://developers.vtex.com/docs/guides/faststore/organisms-image-gallery)</li><li> `__experimentalImageGalleryImage`</li><li> `__experimentalImageGallery`</li><li> `__experimentalShippingSimulation`</li></ul> |
+| Headless CMS: props customization | <ul className="doc-list"><li>ProductTitle: `showDiscountBadge`, `discountBadgeSize`, `showReferenceNumber`</li><li>BuyButton: `Icon`</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
We should add the `__experimentalNotAvailableButton` and `__experimentalProductDescription` too, they're ready to be used as experimental components.

```suggestion
| Overridable components            | <ul className="doc-list"><li>[`ProductTitle`](https://developers.vtex.com/docs/guides/faststore/molecules-product-title)</li><li>[`DiscountBadge`](https://developers.vtex.com/docs/guides/faststore/molecules-discount-badge)</li><li>[`BuyButton`](https://developers.vtex.com/docs/guides/faststore/molecules-buy-button)</li><li>[`Icon`](https://developers.vtex.com/docs/guides/faststore/atoms-icon)</li><li>[`Price`](https://developers.vtex.com/docs/guides/faststore/atoms-price)</li><li>[`QuantitySelector`](https://developers.vtex.com/docs/guides/faststore/molecules-quantity-selector)</li><li>[`SkuSelector`](https://developers.vtex.com/docs/guides/faststore/molecules-sku-selector)</li><li>[`ShippingSimulation`](https://developers.vtex.com/docs/guides/faststore/organisms-shipping-simulation)</li><li>[`ImageGallery`](https://developers.vtex.com/docs/guides/faststore/organisms-image-gallery)</li><li>[`ImageGalleryViewer`](https://developers.vtex.com/docs/guides/faststore/organisms-image-gallery)</li><li> `__experimentalImageGalleryImage`</li><li> `__experimentalImageGallery`</li><li> `__experimentalShippingSimulation`</li><li> `__experimentalNotAvailableButton`</li>`__experimentalProductDescription`</li></ul> |
| Headless CMS: props customization | <ul className="doc-list"><li>ProductTitle: `showDiscountBadge`, `discountBadgeSize`, `showReferenceNumber`</li><li>BuyButton: `Icon`</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```